removeVowelsFromString.py

```diff
@@ -10,14 +10,14 @@
 
 
 #test cases
-case1 = "a"
-case2 = "ab"
-case3 = "g"
-case4 = "aaaaab"
-case5 = "bbbbba"
-case6 = "abababa"
-
-testList = [case1,case2,case3,case4,case5,case5]
+testList = [
+    ("a", ""),
+    ("ab", "b"),
+    ("g", "g"),
+    ("aaaaab", "b"),
+    ("bbbbba", "bbbbb"),
+    ("abababa", "bbb")
+]
```

Comment:
I recommend being explicit about the expected value of the tests. Some other tests to think of adding would be to cover all letters of the alphabet and make sure they have the appropriate behavior. Also would be good to be explicit about upper case and non-alpha (having test cases is a good way to document expected behavior).